scripts/main.js

```diff
@@ -1,5 +1,50 @@
 var helloCork = function() {
-  console.log('Hello Cork');
+  	console.log('Hello Cork');
 };
 
+var signedUrlUpcoming = "https://api.meetup.com/cork-javascript-meetup/events?photo-host=public&page=20&sig_id=210143296&sig=d07ac4460173bc86aa05e0c5b65acd79b2b8ae1b";
+var signedUrlPast = "https://api.meetup.com/cork-javascript-meetup/events?photo-host=public&page=20&sig_id=210143296&status=past&sig=052868f6f4d1f5a525fa6ca28a74e771165ad4ea";
```

Comment:
You can get both past and upcoming events using a request like this: https://api.meetup.com/2/events?offset=0&format=json&limited_events=False&group_urlname=Cork-JavaScript-Meetup&photo-host=public&page=20&fields=&order=time&status=past%2Cupcoming&desc=true

Here's the API docs: https://www.meetup.com/meetup_api/docs/2/events/